api/controllers/routes.go

```diff
@@ -75,7 +75,6 @@ func NewRouter() (router *mux.Router) {
 	router.Handle("/apps/{app}/processes/{process}/run", ws("process.run.attach", ProcessRunAttached)).Methods("GET")
 	router.Handle("/instances/{id}/ssh", ws("instance.ssh", InstanceSSH)).Methods("GET")
 	router.Handle("/proxy/{host}/{port}", ws("proxy", Proxy)).Methods("GET")
-	router.Handle("/services/{service}/logs", ws("service.logs", ServiceLogs)).Methods("GET")
```

Comment:
But I do want to bring this back ASAP. Ideally as part of the provider interface.